[excel] (Samples) Adjust how the samples and scenarios are branded (#686)
OfficeDev/office-scripts-docs: docs/resources/samples/samples-overview.md

---
title: Office Scripts samples
description: Available Office Scripts samples and scenarios.
ms.date: 12/05/2023
ms.localizationpriority: medium
---

# Office Scripts samples and scenarios

This section contains [Office Scripts](../../overview/excel.md) based solutions that help end users achieve automation of daily tasks. It contains realistic scenarios that users face and provides detailed solutions.

> - [Basics](#basics) showcase common examples that make up larger scripts. They are categorized based on a common Excel feature.

Reviewer: ```suggestion
- [Basics](#basics) showcase common examples that make up larger scripts. They're categorized based on a common Excel feature.
```

| Project | Details |
|---------|---------|

> | [Ranges: work with the grid](range-samples.md) | These samples show how to work the `Range` object, which is central to most scripts. |

Reviewer: ```suggestion
| [Ranges: Work with the grid](range-samples.md) | These samples show how to work the `Range` object, which is central to most scripts. |
```

> | [Data validation samples: dropdown lists, prompts, and warning pop-ups](data-validation-samples.md) | These samples show how to use data validation to mandate specific conditions for cell data and how the user is alerted to these rules. |

Reviewer: ```suggestion
| [Data validation samples: Dropdown lists, prompts, and warning pop-ups](data-validation-samples.md) | These samples show how to use data validation to mandate specific conditions for cell data and how the user is alerted to these rules. |
```

| Project | Details |
|---------|---------|
| [Combine worksheets into a single workbook](combine-worksheets-into-single-workbook.md) | This sample uses Office Scripts and Power Automate to pull data from other workbooks into a single workbook. |
| [Convert CSV files to Excel workbooks](convert-csv.md) | This sample uses Office Scripts and Power Automate to create .xlsx files from .csv files. |
| [Cross-reference workbooks](excel-cross-reference.md) | This sample uses Office Scripts and Power Automate to cross-reference and validate information in different workbooks. |
| [Email chart and table images](email-images-chart-table.md) | This sample uses Office Scripts and Power Automate actions to create a chart and send that chart as an image by email. |
| [Record day-to-day changes in Excel and report them with a flow](report-day-to-day-changes.md) | This sample uses a scheduled Power Automate flow to record daily readings and report the changes. |

> | [Run a script on all Excel files in a folder](automate-tasks-on-all-excel-files-in-folder.md) | This project performs a set of automation tasks on all files situated in a folder on OneDrive for Business (can also be used for a SharePoint folder). It performs calculations on the Excel files, adds formatting, and inserts a comment that @mentions a colleague. |

Reviewer 2: "This sample" instead of "this project" to be consistent?

> | [Write a large dataset](write-large-dataset.md) | This sample shows how to send a large range as smaller subranges. |

Reviewer 2: At first glance, I'm not sure it's clear what "send" means and how this scenario fits in the cross-application section?

Author: Fair point. I'll clarify the scenario.

> "This sample shows how to work with a large range as smaller subranges. This helps flows handle data transfer limits and operation timeouts.
